src/protocols/ssh/ttymode.h

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef GUAC_SSH_TTYMODE_H
+#define GUAC_SSH_TTYMODE_H
+
+#include "config.h"
+
+#define TTY_OP_END 0
+#define TTY_OP_VERASE 3
+
+#define GUAC_SSH_TERM_DEFAULT_BACKSPACE 127
+
+char guac_tty_modes[] = {
```

Comment:
Global variables within pre-defined values should be defined in the header file using `extern`, with the value of that variable being provided by a corresponding `.c` file. Assuming this is meant to be a constant, the naming should also follow the `UPPERCASE_WITH_UNDERSCORES` convention being used for constants.

For example, see the definition of `GUAC_TERMINAL_INITIAL_PALETTE` within `palette.h`:

https://github.com/apache/guacamole-server/blob/bc5b01d4d8ab0c3c89a08007316d33012261f6b3/src/terminal/terminal/palette.h#L187-L192

whose value is defined in `palette.c`:

https://github.com/apache/guacamole-server/blob/bc5b01d4d8ab0c3c89a08007316d33012261f6b3/src/terminal/palette.c#L23-L34

Again, assuming this is meant as a constant, it should also be declared with a proper `const` qualifier.

Reply:
Fixed.